packages/ListBox/src/components/RawItem/RawItem.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Option from "../Option";
+/**
+
+This component let the consumer to write `<ListBox.Option preventDefaultOnSelect />`
+in a more semantic way:
+
+<ListBox>
+  <ListBox.Option>Option 1</ListBox.Option>
+  <ListBox.RawItem> this is not interactive and can be anything </ListBox.RawItem>
+</ListBox>
```

Comment:
I don't think inline comments are the best place for this kind of documentation.

Reply:
mmm this is not documentation for our consumers, this is a note for the maintainers, this is a shell component that allowed the consumer to use the `<ListBox.RawItem />` instead of `<ListBox.Option preventDefaultOneSelect />` I think should live here for future reference if someone needs to modify this component. thought if you think there is a better place for this comment I could move it from here. 

Im not attached to the comment, I just want to be clear why this component exists.